app.rb

```diff
@@ -1,4 +1,5 @@
 require 'sinatra/base'
+require 'pry-byebug'
```

Comment:
remove this anywhere else where you require pry. make sure you look over the diff before you create a PR or merge it to master. thats the only way these kind of things gets accidently merged to master.

Reply:
👍